pkg/options/options.go

```diff
@@ -11,9 +14,9 @@ func (l LiteralStringer) String() string {
 //Select prompts the user which
 //of the available options is the desired
 //through STDIN and returns the selected one
-func Select(msg string, opts []fmt.Stringer) interface{} {
+func Select(msg string, opts []fmt.Stringer) (interface{}, error) {
```

Comment:
Does it make sense for `Select` to return an error if we can't do anything about it? Should't we just os.Exit?

Reply:
That will make tests exit rather than error out, I believe. I was running tests on my arm64 device and got an infinite loop since there was no single good match, the test cases expecting the arch to be amd64.

Reply:
> That will make tests exit rather than error out, I believe. I was running tests on my arm64 device and got an infinite loop since there was no single good match, the test cases expecting the arch to be amd64.

Well.. the idea was to loop infinitely and "force" the user select an option (or press ctrl +c to exit). Maybe I had a typo and selected the wrong number and in that case the program will just exit.

Reply:
I can add a check for `io.EOF` which is what `err` becomes in that test situation, but should it then return an error, `nil` or something else?